ToolkitApp/tabs/src/components/sample/MainContent.jsx

```diff
@@ -13,17 +13,23 @@ const MainContent = () => {
     const [thirdSectionActive, toggleThirdSection] = useState(true);
     const [fourthSectionActive, toggleFourthSection] = useState(true);
 
+    const [connectedResourcesList, setResourceList] = useState([]);
+    const [title, setTitle] = useState(" ");
+
+
     let firstSection;
     if (firstSectionActive) {
+        processTeamsContext(setResourceList, connectedResourcesList, setTitle);   
         firstSection =
             <div>
                 <Flex id="main-section" gap="gap.small" onClick={() => toggleFirstSection(!firstSectionActive)}>
                     <ChevronDownIcon id="connected-resource-chevron" />
                     <Header id="connected-resource-header" as="h2" content="Connected Resource" />
                 </Flex>
-                <div>
-                  <p>REMOVE LOREM IPSUM AND PLACE CONNECTED RESOURCES HERE</p>
-                  <p>{dummyText}</p>
+                <div> 
+                    {connectedResourcesList.length !== 0 && <ListItem className = "title" header = {title}/>}
+                    {connectedResourcesList.length !== 0 && <List selectable defaultSelectedIndex={0} items={connectedResourcesList}/>} 
+                    {connectedResourcesList.length === 0 && !title &&  <Alert danger content={NO_CONNECTED_RESOURCES}/>}
```

Comment:
Is there a way to put this inside a component, abstract it away?

Reply:
0cb3efb2c1bdb37baf52d2c9b913480b68c0a205